lib/Persistence/TrafficLimiter.php

```diff
@@ -107,6 +107,39 @@ public static function getHash($algo = 'sha512')
         return hash_hmac($algo, $_SERVER[self::$_ipKey], ServerSalt::get());
     }
 
+    /**
+     * Validate $_ipKey against configured ipranges. If matched ratelimiter will ignore ip
+     *
+     * @access private
+     * @static
+     * @param  string $algo
+     * @return string
+     */
+    private static function matchIp($ipRange = null)
+    {
+        // Match $_ipKey to $ipRange and if it matches it will return with a true
+        $address = \IPLib\Factory::addressFromString($_SERVER[self::$_ipKey]);
+        $range   = \IPLib\Factory::rangeFromString(trim($ipRange));
+        // If $range is null something went wrong (possible invalid ip given in config)
+        if ($range == null) {
+            return false;
+        } else {
+            // Ip-lib does throws and exception when something goes wrong, if so we want to catch it and set contained to false
+            try {
+                $contained = $address->matches($range);
+            } catch (\Exception $e) {
+                // If something is wrong with matching the ip, we set $contained to false
+                return false;
+            }
+        }
+        // Matches return true!
+        if ($contained === true) {
+            return true;
+        } else {
+            return false;
```

Comment:
Also this just return `$contained`.

And remove the comment, it does not explain anything, but actually just repeats what you’ve written in the source code. (Comments should [only] explain _why_ something is written, not what.)